Lib/tkinter/__init__.py

```diff
@@ -34,6 +34,7 @@
 import enum
 import sys
 import types
+import functools
```

Comment:
Can you sort this import?

Reply:
Recommend isort
https://pypi.org/project/isort/

Reply:
No I mean, we already have `import enum, sys, types`. So let's put the `functools` import after the `enum` import.